memtier_benchmark.cpp

```diff
@@ -313,6 +315,8 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
         { "requests",                   1, 0, 'n' },
         { "clients",                    1, 0, 'c' },
         { "threads",                    1, 0, 't' },        
+        { "split",                      0, 0, o_split },
```

Comment:
name is very confusing .. it should be cpu-split or something of this order .. it is unclear what is been split

Reply:
changed to cpu-split